charts/valtimo-backend/templates/deployment.yaml

```diff
@@ -58,6 +58,17 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
       {{- with .Values.nodeSelector }}
+          volumeMounts:
+            - name: media
+              mountPath: {{ .Values.persistence.mountPath }}
```

Comment:
is there a specific reason that this is configurable for the user? As this concerns a mount inside the container which is used by the application, I assume the application expects a specific path? What happens if the user chooses some arbitrary path - will the application be able to handle it?

Reply:
This is by design because the path in the application is also configurable.